pkg/operator/controller.go

```diff
@@ -470,13 +463,23 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 		informers = append(informers, informer)
 	}
 
+	clusterOperatorStatusController := NewClusterOperatorStatusController(
+		c.clients.Config,
+		configInformerFactory.Config().V1().ClusterOperators(),
+		regopInformerFactory.Imageregistry().V1().Configs(),
+		kubeInformerFactory.Apps().V1().Deployments(),
+	)
+
 	configInformerFactory.Start(stopCh)
 	kubeInformerFactory.Start(stopCh)
 	openshiftConfigKubeInformerFactory.Start(stopCh)
 	kubeSystemKubeInformerFactory.Start(stopCh)
 	regopInformerFactory.Start(stopCh)
 	routeInformerFactory.Start(stopCh)
 
+	// TODO(dmage): This controller should be started from main.
+	go clusterOperatorStatusController.Run(stopCh)
```

Comment:
Or from a separate "starter" struct that is effectively main. See https://github.com/openshift/cluster-image-registry-operator/pull/447/commits/eb53b988d47e8a8acfc7f5d00b73bb0fbb19b515

Reply:
Sure. I'd prefer to avoid a struct with one method and have just a function. But it doesn't really matter for me.